admin-dev/themes/new-theme/js/components/auto-complete-search.ts

```diff
@@ -95,11 +128,16 @@ export default class AutoCompleteSearch {
   /**
    * Build the typeahead component based on provided configuration.
    */
-  buildTypeahead() {
-    this.$searchInput.typeahead(this.config, this.dataSetConfig)
-      .bind('typeahead:select', (e, selectedItem) => this.config.onSelect(selectedItem, e, this.$searchInput))
-      .bind('typeahead:close', (e) => {
-        this.config.onClose(e, this.$searchInput);
+  buildTypeahead(): void {
+    /* eslint-disable */
+    this.$searchInput
+      .typeahead(this.config, this.dataSetConfig)
+      .bind('typeahead:select', (e: any, selectedItem: unknown) =>
```

Comment:
```suggestion
      .bind('typeahead:select', (e: any, selectedItem: unknown) =>
```
It's surprising that there is no built-in class for native events

Reply:
There some, but looks like the definition file of typeahead is strange, it doesn't accept `e` as a JQueryEventObject